packages/models-library/src/models_library/projects_state.py

```diff
@@ -80,14 +80,6 @@ class Config:
             ]
         }
 
-    @validator("owner", pre=True, always=True)
-    @classmethod
-    def check_not_null(cls, v, values):
```

Comment:
> not need of owner information if project lock in MAINTAINING state

so what if `status` is not `MAINTAINING`?  What about using a `@root_validator
` that accounts for the values of `onwer` and `status`?

Reply:
improved :+1: